samples/system-test/quickstart.test.js

```diff
@@ -15,44 +15,51 @@
 
 'use strict';
 
-const proxyquire = require(`proxyquire`).noPreserveCache();
+const proxyquire = require(`proxyquire`);
 const sinon = require(`sinon`);
-const test = require(`ava`);
+const assert = require('assert');
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 
-test.before(tools.stubConsole);
-test.after.always(tools.restoreConsole);
+describe('QuickStart', () => {
+	before(() => {
+	  tools.stubConsole;
+	});
+	afterEach(() => {
+	  tools.restoreConsole;
+  });
 
-test.cb(`should query a table`, t => {
-  const databaseMock = {
-    run: _query => {
-      t.deepEqual(_query, {
-        sql: `SELECT 1`,
-      });
-      setTimeout(() => {
-        try {
-          t.deepEqual(console.log.getCall(0).args, [`test`]);
-          t.end();
-        } catch (err) {
-          t.end(err);
-        }
-      }, 200);
-      return Promise.resolve([['test']]);
-    },
-  };
-  const instanceMock = {
-    database: sinon.stub().returns(databaseMock),
-  };
-  const spannerMock = {
-    instance: sinon.stub().returns(instanceMock),
-  };
+  it(`should query a table`, function(done) {
+	this.timeout(15000);
+    const databaseMock = {
+      run: _query => {
+        assert.deepEqual(_query, {
```

Comment:
lint is gonna bite you :) `assert.deepStrictEqual` is probably required here

Reply:
Yes you are right. I did the changes